src/main/java/frc/robot/commands/AutoDriveForward.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+public class AutoDriveForward extends CommandBase {
+    private final Drivetrain m_drivetrain;
+
+    public AutoDriveForward(Drivetrain subsystem) {
```

Comment:
Also, you should pass the distance in through the constructor as a parameter and store it in a field.